git-hub

```diff
@@ -317,67 +387,146 @@ def json_to_toml(username, repo, pr, token, open_or_closed):
     request.add_header('Authorization', 'token %s' % token)
     response = urlopen(request)
     response = json.loads(response.read().decode('utf-8'))
-    to_write = to_write + "['{0} pull requests'.{1}]\n".format(open_or_closed, pr.number)
-    to_write = to_write + 'user = "{0}" \n'.format(pr.user.name)
-    to_write = to_write + 'branch = "{0}" \n'.format(response.get('head').get('ref'))
-    to_write = to_write + 'comment = "{0}" \n'.format(pr.title)
-    to_write = to_write + 'modified = "{0}" \n'.format(pr.updated_at)
-    reviewers = ""
-    for reviewer in list(response.get('requested_reviewers')):
-        reviewers = reviewers + reviewer.get('login')+", "
-    to_write = to_write + 'reviewers = "{0}" \n'.format(reviewers[:-2])
-    assignees = ""
-    for assignee in list(response.get('assignees')):
-        assignees = assignees + assignee.get('login')+", "
-    to_write = to_write + 'assignee = "{0}" \n'.format(assignees[:-2])
-    if(pr.milestone):
-        to_write = to_write + 'milestone = "{0}" \n'.format(pr.milestone.title)
-    else:
-        to_write = to_write + 'milestone = "" \n'
-    issue_url = response.get('issue_url')
+    issue_url = response.get('issue_url')       # fetches labels attached to pull requests
     labels = ""
     issue_request = Request(issue_url)
     issue_response = urlopen(issue_request)
     issue_response = json.loads(issue_response.read().decode('utf-8'))
     labels_dict = issue_response.get('labels')
-    for label in list(labels_dict):
-        labels = labels + label.get('name')+", "
-    to_write = to_write + 'labels = "{0}" \n'.format(labels[:-2])
+    comment_url = response.get('comments_url')       # fetches latest activity (comment) date
+    comment_request = Request(comment_url)
+    comment_response = urlopen(comment_request)
+    comment_dict = json.loads(comment_response.read().decode('utf-8'))\
+
+    reviews = [reviewer.get('login') for reviewer in list(response.get('requested_reviewers'))]
+    assignees = [assignee.get('login') for assignee in list(response.get('assignees'))]
+    labels = [label.get('name') for label in list(labels_dict)]
+    comment_dates = [c.get('updated_at') for c in list(comment_dict)]
+
+    to_write = f""" 
```

Comment:
Looks much better!

To prevent indentation problems:

```
import textwrap
textwrap.dedent(to_write)
```
That way, you can indent the code as much as needed, and it still writes to file properly.